src/main/java/com/nitnelave/CreeperHeal/block/CreeperBanner.java

```diff
@@ -1,16 +1,19 @@
 package com.nitnelave.CreeperHeal.block;
 
+import com.nitnelave.CreeperHeal.config.CreeperConfig;
+import org.bukkit.Bukkit;
 import org.bukkit.block.Banner;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.BannerMeta;
 
 /**
  * Banner implementation of CreeperBlock.
- * 
- * @author drexplosionpd
```

Comment:
Add yourself to the authors, don't remove the previous guy.

Reply:
This was actually a [deletion](https://github.com/nitnelave/CreeperHeal/pull/60/commits/42787f31159949b2eaec5bce05e59abe03c3e079#diff-8a8069a7b97a978f01aca8ac4686c101) and [implementation](https://github.com/nitnelave/CreeperHeal/pull/60/commits/f3b468bd793ea17c8393ccde13effe27c8dfc544#diff-8a8069a7b97a978f01aca8ac4686c101). It's not required to special case banners to restore them, which was why this was deleted, that's all it does. However, the bug I found later requires banners to be special cased for dropping the correct item, so I added my own implementation. Looks bad considering the diffs of all the commits as a whole, but that's how we ended up where we are. Can re-add, no problem.

Reply:
I'd like you to keep the previous author, if it's okay :)